packages/mobile/navigation/AppNavigator.tsx

```diff
@@ -2,49 +2,84 @@ import React from "react";
 import HomeScreen from "../screens/Home";
 import { createDrawerNavigator } from "@react-navigation/drawer";
 import { createStackNavigator } from "@react-navigation/stack";
-
-import {
-  Image,
-  ScrollView,
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View
-} from "react-native";
-import { Button } from "react-native-paper";
+import { IconButton } from "react-native-paper";
+import { View } from "react-native";
+import ArticleContent from "../components/ArticleContent";
```

Comment:
Dont add components straight to the navigator, we should have an article screen that uses the article editor